tests/functional/vault/test_misc.py

```diff
@@ -124,14 +124,38 @@ def test_negative_available_deposit_limit(Vault, token, gov):
 
 
 def test_sweep(gov, vault, rando, token, other_token):
-    token.transfer(vault, token.balanceOf(gov), {"from": gov})
-    other_token.transfer(vault, other_token.balanceOf(gov), {"from": gov})
+    # Vault wrapped token reverts on 0 sweep
+    with brownie.reverts():
+        vault.sweep(token, {"from": gov})
 
-    # Vault wrapped token doesn't work
+    # Airdrop want token
+    airdropAmount = token.balanceOf(gov) // 2
```

Comment:
can we add a similar test outside sweep where we assert an airdrop doesn't change the state of PricePerShare so we have it specd as test